src/lib/payload.ts

```diff
@@ -1,4 +1,5 @@
-import { CertificateRow } from './db'
+import { AttachmentRow, CertificateRow } from './db/types'
```

Comment:
DB changes as separate PR from API controller, testing first with initiation and issuance?